server/athenian/api/models/web/contributor.py

```diff
@@ -0,0 +1,130 @@
+from typing import Optional
+
+from athenian.api.models.web.base_model_ import Model
+
+
+class Contributor(Model):
+    """Contributor details."""
```

Comment:
If this is a spec description then we should be a bit more verbose here.
```suggestion
    """Details about a developer who contributed to some repositories owned by the account."""
```

Reply:
added both here and in spec